apps/judicial-system/api/src/app/modules/police/models/policeCaseFile.model.ts

```diff
@@ -13,6 +13,9 @@ export class PoliceCaseFile implements TPoliceCaseFile {
   @Field()
   readonly policeCaseNumber!: string
 
-  @Field()
-  readonly displayDate!: string
+  @Field({ nullable: true })
+  readonly chapter?: number
+
+  @Field({ nullable: true })
```

Comment:
I thought this date field was non nullable, i.e. I thought it was always set. Is there a reason we are making it optional?

Reply:
Prombem is, we don't know. The data from LÖKE may or may not contain these new fields and by keeping them optional we avoid potential runtime errors.